vignettes/baselinenowcast.Rmd

```diff
@@ -114,24 +106,29 @@ at each reference date, observed as of the "fully observed" data on October
 ggplot() +
   geom_line(
     # Plot the data summed across reporting days
-    data = observed_long |>
-      group_by(reference_date) |>
-      summarise(total_confirmed = sum(confirm)),
-    aes(x = reference_date, y = total_confirmed), color = "darkred"
+    data = epinowcast::germany_covid19_hosp |>
+      enw_filter_report_dates(latest_date = "2021-07-01") |>
+      dplyr::filter(
+        location == "DE", age_group == "00+",
+        report_date == "2021-07-01"
+      ),
+    aes(x = reference_date, y = confirm), color = "darkred"
   ) +
   geom_line(
     data = epinowcast::germany_covid19_hosp |>
-      dplyr::filter(location == "DE", age_group == "00+") |>
-      group_by(reference_date) |>
-      summarise(total_confirmed = sum(confirm)) |>
-      dplyr::filter(reference_date <= "2021-07-01"),
-    aes(x = reference_date, y = total_confirmed), color = "black"
+      dplyr::filter(
```

Comment:
I dont think we need to use package::function here its certainly not epinowcast org standards